src/components/Editor/CodeEditor.css

```diff
@@ -18,3 +18,7 @@
   user-select: none;
   cursor: default;
 }
+
+.CodeMirror-lines {
+  padding-top: 24px;
+}
```

Comment:
`CodeMirror-lines` corresponds to the line area in the editor. Since we have to put the padding-top all over the editor, how about putting it in `CodeMirror`?

`CodeMirror-lines`
<img width="563" alt="Screen Shot 2020-11-29 at 4 07 25 PM" src="https://user-images.githubusercontent.com/2059311/100535564-77e86b80-325d-11eb-9c28-869f8e28b474.png">

`CodeMirror`
<img width="413" alt="Screen Shot 2020-11-29 at 4 07 46 PM" src="https://user-images.githubusercontent.com/2059311/100535576-89ca0e80-325d-11eb-8838-7301b4eac856.png">

Reply:
It seems like space at the top of the editor is wasted. If it is at least 16px, I have confirmed that the GNB and the nameplate do not overlap. How about reducing it to 16px or 18px?

Reply:
> CodeMirror-lines corresponds to the line area in the editor. Since we have to put the padding-top all over the editor, how about putting it in CodeMirror?

good 😄

Reply:
> It seems like space at the top of the editor is wasted. If it is at least 16px, I have confirmed that the GNB and the nameplate do not overlap. How about reducing it to 16px or 18px?

I set the spacing a little bit loose, but It will be set to 18px, and if it's strange, change it again.

Reply:
> > CodeMirror-lines corresponds to the line area in the editor. Since we have to put the padding-top all over the editor, how about putting it in CodeMirror?
> 
> good 😄

When I put the padding value here `CodeMirror`, it is hidden. How did you test it?? 🧐

Reply:
It would be nice to add the reason for putting the padding-top as a comment.

Reply:
@hackerwins Thank you for your review. 

There is a hidden style applied to the `CodeMirror-scroll` inside `Codemirror`, so I think I should set the padding in `CodeMirror-lines`.  I wrote a comment. https://github.com/yorkie-team/yorkie-codepair/pull/17/commits/d6a3f9f0bc81f8b62cee89bc5d6c57e59dfa8ab1#diff-d2824b99bb096857de1c550bd74df23f40d04255460741532be1e4d680b168afR22

Reply:
I See. It would be better to apply it to `CodeMirror-scroll`.